packages/plugin-selenium-driver/src/plugin/index.ts

```diff
@@ -288,6 +288,15 @@ export class SeleniumPlugin implements IBrowserProxyPlugin {
         }
     }
 
+    public async newWindow(applicant: string, val: string, windowName: string, windowFeatures: string) {
```

Comment:
`openWindow`, maybe? We don't use exactly webdriver.io API, so it's better to use more friendly names.

Reply:
Ok, I will rename to openWindow